src/Agent.Plugins/Artifact/FileContainerProvider.cs

```diff
@@ -143,11 +148,41 @@ private async Task DownloadFileContainerAsync(Guid projectId, BuildArtifact arti
                 new ExecutionDataflowBlockOptions()
                 {
                     BoundedCapacity = 5000,
-                    MaxDegreeOfParallelism = 8,
+                    MaxDegreeOfParallelism = downloadParameters.ParallelizationLimit,
```

Comment:
curious why we are adding the option to parallelize

Reply:
The current build artifacts task has a parallelism option. This is for maintaining compat with that option.